src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Updated [Http Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/specification/trace/semantic_conventions/http.md).
```

Comment:
Probably better to point to a particular commit/tag so instead of main.

Reply:
The new repo doesn't have any tags yet! 😭
I'll get the commit url.

Reply:
updated